projects/plugins/jetpack/modules/contact-form/css/grunion.css

```diff
@@ -16,14 +16,22 @@
 	opacity: 0.3;
 }
 
-.contact-form input[type='text'],
-.contact-form input[type='email'],
-.contact-form input[type='tel'],
-.contact-form input[type='url'] {
+/*
+	Using :where we will keep a lower CSS specificity,
+	allowing themes to easily override these styles
+ */
+:where(
+	.contact-form input[type='text'],
+	.contact-form input[type='email'],
+	.contact-form input[type='tel'],
+	.contact-form input[type='url']
+) {
```

Comment:
Is this addressing a known issue? I'm not too up-to-date regarding the use of `:where`, but unless you feel really comfortable with it, I'd say we need to check we're not breaking any form/theme implementation out there.

The style change works nicely, I'll leave it up to you regarding the (pseudo)selector change.

Reply:
Some themes themes override the input styles using a one level selector, like `input[type='text'] { ... }`.
Using `.contact-form input[type='text']` makes these styles more specific than the theme's styles and it's causing some inconsistencies.
I changed it to use `:where` to keep a low specificity.